doc/modules/changes.h

```diff
@@ -6,6 +6,12 @@
  *
  *
  * <ol>
+ * 
+ * <li> New: There are now postprocessors BoundaryDensities and
+ * BoundaryPressures which calculate laterally averaged densities 
+ * and pressures at the top and bottom of the domain. The intention
+ * is that they may be used for dynamic topography calculations.
+ * (Ian Rose, 2015/05/27)
```

Comment:
Drop the last sentence. People come up with all sorts of reasons why they want to use stuff, no need to limit their imagination :-)